src/components/NotificationIcon/NotificationIcon.tsx

```
@@ -0,0 +1,87 @@
+                notification.navigation && navigate(notification.navigation)
+              }
+              style={{
+                cursor: notification.navigation ? 'pointer' : 'default',
+              }}
+            >
+              {!notification.isRead && (
+                <span className={styles.notificationDot} title="Unread" />
+              )}
+              <span className={styles.notificationText}>
+                {notification.body.length > 48
+                  ? notification.body.slice(0, 48) + '...'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)